.github/workflows/sdk.yml

```diff
@@ -285,6 +285,19 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
 
+      - name: generate python coverage report
+        working-directory: python
+        run: |
+          pip install coverage
+          coverage run -m unittest
+
+      - name: upload python coverage to codecov
+        uses: codecov/codecov-action@v2
+        with:
+          name: python coverage
```

Comment:
`files` options is needed, so codecov can parse and upload the coverage stats

Reply:
Done